plugins/views/frontend/public/javascripts/countly.views.js

```diff
@@ -1,797 +1,438 @@
                 }
-                var apiQueryData = {
-                    api_key: countlyGlobal.member.api_key,
-                    app_id: countlyCommon.ACTIVE_APP_ID,
-                    path: requestPath,
-                    method: "GET",
-                    filename: "Views" + countlyCommon.ACTIVE_APP_ID + "_on_" + moment().format("DD-MMM-YYYY"),
-                    prop: ['aaData'],
-                    "url": "/o/export/requestQuery"
-                };
-                return apiQueryData;
-            }
-            return null;
-        },
-        renderCommon: function(isRefresh) {
-            var self = this;
-            var props = this.getProperties();
-            var usage = [];
-            this.segmentMap = countlyViews.getSegments();
-            var i;
-
-            for (i in props) {
-                usage.push({
-                    "title": props[i],
-                    "id": "view-metric-" + i
-                });
-            }
-
-            var domains = countlyViews.getDomains();
-            for (i = 0; i < domains.length; i++) {
-                domains[i] = countlyCommon.decode(domains[i]);
-            }
-
-            this.templateData = {
-                "page-title": jQuery.i18n.map["views.title"],
-                "font-logo-class": "fa-eye",
-                "active-segmentation": jQuery.i18n.map["views.all-segments"],
-                "segmentations": countlyViews.getSegmentKeys(),
-                "usage": usage,
-                "domains": domains
-            };
-            if (self.selectedSegment.segmentKey !== "") {
-                this.templateData["active-segmentation"] = self.selectedSegment.segmentKey;
-                this.templateData["segment-values"] = self.segmentMap[self.selectedSegment.segmentKey];
-                if (self.selectedSegment.segmentValue !== "") {
-                    this.templateData["active-segmentation-value"] = self.selectedSegment.segmentValue;
+                this.isUpdateButtonDisabled = good;
+            },
+            deleteManyViews: function() {
+                if (this.selectedViews.length > 0) {
+                    if (this.selectedViews.length === 1) {
+                        this.deleteDialogTitle = CV.i18n('views.delete-confirm-title');
+                        this.deleteDialogText = CV.i18n('views.delete-confirm').replace("{0}", this.selectedViews[0].display);
+                        this.deleteDialogConfirmText = CV.i18n('views.yes-delete-view');
+                    }
+                    else {
+                        var names = [];
+                        for (var k = 0; k < this.selectedViews.length; k++) {
+                            names.push(this.selectedViews[k].display);
+                        }
+                        this.deleteDialogTitle = CV.i18n('views.delete-many-confirm-title');
+                        this.deleteDialogText = CV.i18n('views.delete-confirm-many').replace("{0}", names.join(", "));
```

Comment:
you can use the jQuery.i18n.prop notatation here :

`CV.i18n('views.delete-confirm-many',  names.join(", "))`

instead of

`CV.i18n('views.delete-confirm-many').replace("{0}", names.join(", "))`